package.json

```diff
@@ -25,11 +25,12 @@
   },
   "scripts": {
     "build": "NODE_ENV=production CI=false && npm-run-all build:*",
-    "build:css": "NODE_ENV=production postcss src/tailwind.css -o build/index.css",
+    "build:tw-config": "cpy 'src/tailwind.js' build",
+    "build:css": "NODE_ENV=production postcss src/tailwind.css -o build/styles.css",
```

Comment:
u think can remove this build:css liao?

Reply:
think we can leave it, its more for say ppl who want to use common ui + but not gonna setup tailwind at all, so they can import the css directly and the common ui should still work. whether there is conflicts then, that's out of our scope already